camelot/core/orm/entity.py

```diff
@@ -371,15 +371,23 @@ def get_cls_by_discriminator(cls, discriminator_value):
                 if groups and primary_discriminator in types.__members__ and types[primary_discriminator].grouped_by is not None:
                     group = types[primary_discriminator].grouped_by.name
 
-                # Support passed secondary discriminator arguments both on the instance as the class level.
+                # Support passing secondary discriminator arguments both on the instance as the class level.
```

Comment:
mss in deze method van 'discriminator_value' altijd tuples maken, zodat we nt moeten kijken naar welk type het argument is